File: solutions/cpp/darts/1/darts.cpp
#include "darts.h"

namespace darts {

// TODO: add your solution here
    int score(double x, double y){
        double sum = x * x + y * y;
        if (sum <= 1.0)
            return 10;
        else if (sum > 1.0 && sum <= 25.0)
            return 5;
        else if (sum > 25.0 && sum <= 100.0)
            return 1;
        return 0;
    }
}  // namespace darts

File: solutions/python/darts/1/darts.py
def score(x, y):
    if (distanceFromOrigin(x, y)) > 10:
        return 0
    if 5 < distanceFromOrigin(x, y) <= 10:
        return 1
    if 1 < distanceFromOrigin(x, y) <= 5:
        return 5
    if distanceFromOrigin(x, y) <= 1:
        return 10
    return None
def distanceFromOrigin(x, y):
    return (x * x + y * y) ** 0.5